dell/src/main/java/org/apache/iceberg/dell/DellProperties.java

```diff
@@ -38,6 +40,21 @@
    */
   public static final String ECS_S3_ENDPOINT = "ecs.s3.endpoint";
 
+  /**
+   * Catalog prefix is used to store catalog data. If not set, use {@link CatalogProperties#WAREHOUSE_LOCATION}.
+   * <p>
+   * The value is an EcsURI which like ecs://bucket/prefix.
+   */
+  public static final String ECS_CATALOG_PREFIX = "ecs.catalog.prefix";
```

Comment:
As we usually specify a warehouse location in the FS based catalog,  why do we still need an extra `ecs.catalog.prefix` config which has the similar meaning as the `CatalogProperties#WAREHOUSE_LOCATION` ?

Reply:
If the customer needs to separate the data and catalog metadata, that can set this config. 
This config is not required. The default value is `CatalogProperties#WAREHOUSE_LOCATION`.

Reply:
I still don't understand what's the benefit that we made the metadata into a customized separate path.

Reply:
If you really want to make a separate catalog path for the ECS metadata,  I will suggest to name it as `ecs.catalog.metadata.path` . The `ecs.catalog.prefix` is quite confuse for me, why do the catalog need a prefix ?